homeassistant/components/sensor/sht31.py

```diff
@@ -0,0 +1,112 @@
+"""
+Support for Sensirion SHT31 temperature and humidity sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sht31/
+"""
+
+import logging
+import math
+import voluptuous as vol
```

Comment:
I'm not sure if this is required for PRs, but pointing it out anyway: In Home Assistant we usually keep imports grouped in the following order:

1. Python internal libraries.
2. External global libraries.
3. Home Assistant local imports.

So your imports would (for consistency) look like this:

```python
import logging
import math

import voluptuous as vol

from homeassistant.components.sensor import PLATFORM_SCHEMA
# ...
```